src/components/search/Autocomplete.tsx

```diff
@@ -28,10 +29,17 @@ export function Autocomplete(props) {
       ...props,
     });
 
+    const inputElement = containerRef.current.querySelector("input");
+    if (!inputElement || inputElement.value === "") {
```

Comment:
This if condition seems wrong plz check again.
when inputElement is null you are trying to focus it